pipelines/build/common/build_base_file.groovy

```diff
@@ -908,8 +908,9 @@ class Builder implements Serializable {
                 }
             } else if (publish && release) {
                 context.println "NOT PUBLISHING RELEASE AUTOMATICALLY"
-            } else if (release ) {
+            } else if (release && enableTests) {
                 //remote trigger job https://ci.eclipse.org/temurin-compliance/job/AQA_Test_Pipeline/
+                context.echo "Enable tests: ${enableTests}"
```

Comment:
```suggestion
                context.echo "Enable tests: ${enableTests} so we will trigger the remote JCK jobs"
```

Reply:
Thanks @karianna ! That's originally added as testing code and forgot to update. Updated!